artiq/firmware/libboard_artiq/hmc830_7043.rs

```diff
@@ -113,7 +113,8 @@ mod hmc830 {
                     if addr == 0x0d || addr == 0x0e { continue; }
                     error!("[0x{:02x}] = 0x{:04x}", addr, read(addr));
                 }
-                return Err("HMC830 lock timeout");
+            //    return Err("HMC830 lock timeout");
```

Comment:
Patches should do a single thing, so disabling the HMC830 should not be here.

Reply:
Sorry, that wasn't supposed to be there.